contrib/builder/deb/armhf/raspbian-jessie/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM resin/rpi-raspbian:jessie
```

Comment:
@luxas Thanks for the pointer. I haven't noticed these Docker images. I'll try that.